### ISSUE TITLE
Hit markers larger

### ISSUE THREAD

**Issue by deferred-impact** [deferred-impact/battledudes-issue-tracker#225]
Many skins hide hit markers

E.g. the space helmet

I can't see shit for fuck's sake

**Comment by ghost** [comment]
Eh, not that big of an issue.

**Comment by deferred-impact** [comment]
> Eh, not that big of an issue.

As big an issue as #231 , lmao

**Comment by ghost** [comment]
Nah, it isn't. Seeing if someone is hit by a bullet, isn't more important than being able to see whether someone is cooking a weapon that will one shot you. 
Besides, hit markers are useless with guns, it's obvious when a bullet hits them.

**Comment by deferred-impact** [comment]
> Nah, it isn't. Seeing if someone is hit by a bullet, isn't more important than being able to see whether someone is cooking a weapon that will one shot you.

Well i'll concede to this, a single Flintlock shot is less important than a cooked grenade, inherently. But both of these are important sources of gameplay information, so it's not unreasonable to treat them similarly.

> Besides, hit markers are useless with guns, it's obvious when a bullet hits them.

Wow, tell me you never used a sniper weapon without explicitly saying you never used a sniper weapon.

Hit markers are a necessity, they somewhat mitigate the issue of desync. Plenty of times you'll see your HS shot hit the enemy, but no hit marker will appear. Sucks, but at least you know you just got desynced, and can react accordingly.

**Comment by ghost** [comment]
> Well i'll concede to this, a single Flintlock shot is less important than a cooked grenade, inherently. But both of these are important sources of gameplay information, so it's not unreasonable to treat them similarly.

If you are hit by a flintlock shot, and don't see the marker:
A. You notice that it hits you, even if you don't see the markers, because you lose some health
B. You don't die to it instantly, if at full health
C. You can notice that it hit you, even if you don't look at your health or the marker, because you can see that the bullet clearly collided with your body (in game)

On the damage inflicting side, you can also see, that the bullet hit the player, hit marker or not. 

If you get hit by a grenade, and don't see it being cooked:
A. You die instantly
B. There's no other way for you to have seen it coming

Now I'll assume you're bright enough to put 2 and 2 together, and understand my point. 

> Hit markers are a necessity, they somewhat mitigate the issue of desync. Plenty of times you'll see your HS shot hit the enemy, but no hit marker will appear. 

To be honest, I've never experienced something like that, so I cannot say much about it. But I can say though, that having the knowledge about your desync shot, is not as primary as knowing if you'll be one-shotted in the next 3 seconds, by someone cooking a grenade under their concealing hat. Thus I think the issue I posted is a bigger issue, because my issue was actually 'inspired' by yours. 

> and can react accordingly.

How, precisely?
What strategic motion would a player have to plan after knowing that one of their shots has been desynced? You would just still continue shooting wouldn't you?
Your next shot, should let you know whether you hit it the first time or not. Besides, bullet trajectory is predictable enough to let someone know if they hit the enemy or not.

**Comment by deferred-impact** [comment]
> On the damage inflicting side, you can also see, that the bullet hit the player, hit marker or not.

Dude. **DESYNC.** Just because you saw the bullet hit the enemy, doesn't mean it dealt damage.

> Now I'll assume you're bright enough to put 2 and 2 together, and understand my point.

Actually no idea what you were trying to say. That the grenade cooking is more important than the hit markers? Well i literally agreed in my previous comment with that

> But I can say though, that having the knowledge about your desync shot, is not as primary as knowing if you'll be one-shotted in the next 3 seconds, by someone cooking a grenade under their concealing hat

I literally admitted this in the last comment, dude...

> Your next shot, should let you know whether you hit it the first time or not. Besides, bullet trajectory is predictable enough to let someone know if they hit the enemy or not.

Dude, how many hours have you played with HeavySniper? You've got to be kidding. You act like you never used it.

This statement in particular made zero sense. "should let you know whether you hit it the first time or not."? What?

Knowing if the bullet dealt damage or not, and knowing the enemy's health in general, is a major and very important factor in combat

You say the dumbest shit at times. This last comment of yours is the perfect example.

**Comment by ghost** [comment]
> Dude. DESYNC. Just because you saw the bullet hit the enemy, doesn't mean it dealt damage.

dude, you like, *love* making things sound like they happen way more often than they really do lmfao

On average, desync happens like, 1/200 bullets fired, and that too, if the bullet is nearly at the end of its range, which means in 9/10 cases, you will hit the shot. 

"GRRGRGR WHAT ABOUT THE 1 CASE WHERE IT DOESN'T YOYO!!!11!11"

You can learn if it hits the player by using logic: In your next shot, if they die, you hit the previous one. 
If they don't, it was desync. 

"GRRGGAGARA WHAT IF THEY ESCAPE THO GRGRGRGR"

Doesn't matter in that situation. Whether you hit your shot or not, if they run away to heal, then they heal, no matter if desync happened or not. 

> Actually no idea what you were trying to say. That the grenade cooking is more important than the hit markers? Well i literally agreed in my previous comment with that

See? You're smart enough to understand it. 
No point of this discussion if you get my simple point. 

> Dude, how many hours have you played with HeavySniper? You've got to be kidding. You act like you never used it.

Not relevant. 

> This statement in particular made zero sense. "should let you know whether you hit it the first time or not."? What?

You seriously need me to spoonfeed you everything?

I see a guy with oversized hat
I hit a guy with HS
he runs away
I hit him again with sniper

A. if he dies, then the first shot was hit
B. If he doesn't die, then desync


> Knowing if the bullet dealt damage or not, and knowing the enemy's health in general, is a major and very important factor in combat

Thankyou for enlightening me with your words of wisdom kind sir. Now listen to my words. 

A. Doesn't matter if desync forces your shot to be hit or not, vs a nade, it is not the priority
B. Skins don't hide the hit marker as well, **because they aren't oversized from all angles,** they still can be seen from the back of the skin, or from the corners. Its the front of a cap or helmet that is oversized, and most importantly, **hold the potential to hide cooked grenades.**
C. If you admit nades are more of a priority than a bullet then there is no need for this argument.

**Comment by deferred-impact** [comment]
> Not relevant.

Hilarious dodge of a question. How many hours have your played HeavySniper?

> A. if he dies, then the first shot was hit
B. If he doesn't die, then desync

The strategy depends heavily on how much health the enemy has. Knowing how much health the enemy has is incredibly useful, and hit markers are the way to get that info

> Skins don't hide the hit marker as well, because they aren't oversized from all angles,

Plenty of skins are

>  C. If you admit nades are more of a priority than a bullet then there is no need for this argument.

Then why are you still arguing

**Comment by ghost** [comment]
> Hilarious dodge of a question. How many hours have your played HeavySniper?

What, you think I was counting minutes when I was playing it?
There's no stat for that, none that I know of. 

> The strategy depends heavily on how much health the enemy has. Knowing how much health the enemy has is incredibly useful, and hit markers are the way to get that info

? Everyone has the same health. Even if Kev, a HS can still two shot. And how exactly do hit markers allow you to tell someone's health? They only tell if someone is hit. 

> Plenty of skins are

Ok, name one that covers hit markers from all angles and sides. 

> Then why are you still arguing

Fine. I won't respond anymore.

**Comment by deferred-impact** [comment]
> What, you think I was counting minutes when I was playing it?
There's no stat for that, none that I know of.

You should be able to say an approximate amount. There's a difference between never playing it, playing it for 30 minutes, playing it for 5 hours, playing it for 10 days.

> ? Everyone has the same health. Even if Kev, a HS can still two shot. And how exactly do hit markers allow you to tell someone's health? They only tell if someone is hit.

This is the dumbest sentence i read this whole week, and that's saying a lot, considering.

> Ok, name one that covers hit markers from all angles and sides.

The full hazmat suit one. The lifebuoy. The red samurai hat. Tons of other examples

**Comment by ghost** [comment]
> You should be able to say an approximate amount. There's a difference between never playing it, playing it for 30 minutes, playing it for 5 hours, playing it for 10 days.

idk, never kept track, and I switch my gear very very often, so I can't give you a close guess. 
I can give you kills if you want. 

> The full hazmat suit one. The lifebuoy. The red samurai hat. Tons of other examples

I believe the front of the hazmat shows hit markers, and slightly from the back too. 
Lifesaver and Square robot might be an exception. 
Samurai shows from front. 

Simple solution to this: just increase size of hit markers. 
Still, my original point exists, making sure nades are visible should be a bigger priority than bullets. 
Though it doesn't matter anyways, its not like the developer is smart enough to implement any of this.

**Comment by deferred-impact** [comment]
>  I can give you kills if you want.

That would be a decent metric, sure

> Simple solution to this: just increase size of hit markers.

Wow, it's almost as if this is what the issue is about

**Comment by ghost** [comment]
> That would be a decent metric, sure

![Screenshot 2023-10-16 4 32 06 PM](https://github.com/deferred-impact/battledudes-issue-tracker/assets/148166198/b4343817-486e-4f7b-b45a-fd6d5e03690a)

This a screenshot of a couple days ago. Why not current? Because I'm too lazy to go and check the current amount of kills, and take a screenshot. 

> Wow, it's almost as if this is what the issue is about

Nooooooo, ofc not.....

![image](https://github.com/deferred-impact/battledudes-issue-tracker/assets/148166198/8ce6189f-9fba-4997-9d60-3aaa203dad94)

Its not like that's the issue, now, is it?

**Comment by deferred-impact** [comment]
Well. I'm incredibly surprised that a HeavySniper user with 15k kills does not consider desync to be a massive, game-ruining issue.

**Comment by ghost** [comment]
No, I've never seen it as a game ruining, absolutely game corrupting issue. I've hardly even experienced anything like desync, so idk.
